webpack.prod.config.js

```diff
@@ -13,6 +15,17 @@ function resolvePrivateEnvConfig(filePath) {
 }
 resolvePrivateEnvConfig('.env.private');
 
+const plugins = []
+// PurgeCSS
+if (process.env.USE_PURGECSS) {
+  plugins.push(
+    new PurgeCSSPlugin({
+      paths: [glob.sync('node_modules/@edx/paragon/**/*', {nodir: true})],
+      whitelistPatterns: [/pgn[-_]/]
```

Comment:
I don't see this as an option to pass to `PurgeCSSPlugin` anymore; did you mean `safelist` ([source](https://github.com/FullHuman/purgecss/tree/main/packages/purgecss-webpack-plugin#safelist))?